Formula/tarantool.rb

```diff
@@ -46,12 +45,27 @@ def install
     args << "-DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}"
     args << "-DREADLINE_ROOT=#{Formula["readline"].opt_prefix}"
     args << "-DENABLE_BUNDLED_LIBCURL=OFF"
-    args << "-DCURL_INCLUDE_DIR=#{sdk}/usr/include"
-    args << "-DCURL_LIBRARY=#{sdk}/usr/lib/libcurl.#{lib_suffix}"
-    args << "-DCURSES_NEED_NCURSES=TRUE"
-    args << "-DCURSES_NCURSES_INCLUDE_PATH=#{sdk}/usr/include"
-    args << "-DCURSES_NCURSES_LIBRARY=#{sdk}/usr/lib/libncurses.#{lib_suffix}"
-    args << "-DICONV_INCLUDE_DIR=#{sdk}/usr/include"
+    args << "-DENABLE_BUNDLED_LIBYAML=OFF"
```

Comment:
I don't recommend building with system libyaml: there are subtle behaviour differences. See https://github.com/tarantool/tarantool/issues/4090 for details.

Reply:
Using of a system Zstd should be okay, though.

Reply:
I think bundled libyaml still compiled at least (bundled zstd definitely does not on arm64 macOS) so I could switch that one back, but is there a reason any patch to libyaml has not been upstreamed? Avoiding bundled libraries isn't really an uncommon philosophy in package managers in general there should be a documented reason why a forked libyaml is necessary.

Reply:
We didn't propose this change to the upstream, because of doubts that libyaml developers will happy with the libicu dependency, while we depend on it anyway.

Sure, we should create documentation for downstream maintainers.

Reply:
We have arm64 Mac OS 11.2 in CI and the build with bundled zstd works good. Strange thing.

Anyway, I agree, distro maintainers shoud generally prefer system libraries.